modules/ceph/src/test/java/org/testcontainers/containers/CephContainerTest.java

```diff
@@ -0,0 +1,112 @@
+package org.testcontainers.containers;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.Bucket;
+import com.amazonaws.services.s3.model.S3Object;
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CephContainerTest {
+    @Test
+    public void testS3Bucket() {
+        try (
+            // creating_container {
+            CephContainer cephContainer = new CephContainer()
+                .withAwsAccessKey("test")
+                .withAwsSecretKey("test")
+                .withBucketName("test");
+            // }
+        ) {
+            cephContainer.start();
+
+            final AmazonS3 amazonS3 = buildS3Client(cephContainer);
+
+            String bucketName = "test2";
+
+            //check current bucket
+            assertTrue(amazonS3.doesBucketExistV2(cephContainer.getBucketName()));
+
+            //create another bucket
+            amazonS3.createBucket(bucketName);
+            assertTrue(amazonS3.doesBucketExistV2(bucketName));
+            List<Bucket> buckets = amazonS3.listBuckets();
+            assertEquals(2, buckets.size());
+            assertTrue(buckets.stream().anyMatch(
+                bucket -> bucket.getName().equals(bucketName)
+            ));
+
+            //remove bucket
+            amazonS3.deleteBucket(bucketName);
+            assertFalse(amazonS3.doesBucketExistV2(bucketName));
+            buckets = amazonS3.listBuckets();
+            assertEquals(1, buckets.size());
+            assertFalse(buckets.stream().anyMatch(
+                bucket -> bucket.getName().equals(bucketName)
+            ));
+        }
+    }
+
+    @Test
+    public void testS3Object() throws IOException {
```

Comment:
perhaps we don't need both tests (or at least should make the container static + `@ClassRule` to avoid having to start it twice)